snowflake/udfs/udfs/traffic_drop/traffic_drop_udf.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+import uuid
+
+import pandas as pd
+
+class Result:
+    def __init__(self, job_type, detectionID, endpoint, direction, avgDrop, stdevDrop, anomalyDropDate, anomalyDropNumber):
+        self.job_type = job_type
+        if not detectionID:
+            self.detectionID = str(uuid.uuid4())
+        else:
+            self.detectionID = detectionID
+        self.time_created = datetime.datetime.now()
+        self.endpoint = endpoint
+        self.direction = direction
+        self.avgDrop = avgDrop
+        self.stdevDrop = stdevDrop
+        self.anomalyDropDate = anomalyDropDate
+        self.anomalyDropNumber = anomalyDropNumber
+
+class TrafficDrop:
+    def __init__(self):
+        self._date_dropnumber_pairs = []
```

Comment:
`_dateDropNumberPairs` for consistency?

please either use snake_case or camelCase for all the Python variables names. Which convention did we follow for the NP rec Python UDTF?

Reply:
Thanks Antonin, I agree with you we should use a consistent naming convention for our Python variables. I think snake_case aligns better with the Python naming convention  while our data schema uses camelCase currently. This mixed usage has occurred in the NP rec and I have been trying to use camelCase for arguments passed from the snowflake table and snake_case for internal Python variables.
To eliminate ambiguity, I plan to implement snake_case for all function arguments and variables going forward. Would that be acceptable to you?

Reply:
sounds good, let's make it all snake_case